api/v1alpha1/sandbox_types.go

```diff
@@ -66,6 +66,11 @@ type SandboxSpec struct {
 	// PodTemplate describes the pod spec that will be used to create an agent sandbox.
 	// +kubebuilder:validation:Required
 	PodTemplate PodTemplate `json:"podTemplate" protobuf:"bytes,3,opt,name=podTemplate"`
+
+	// ShutdownTime - Absolute time (RFC 3339) when the sandbox is deleted.
+	// If a time in the past is provided, the sandbox will be deleted immediately.
+	// +kubebuilder:validation:Format="date-time"
+	ShutdownTime *metav1.Time `json:"shutdownTime,omitempty"`
```

Comment:
I was debating shutdownTime vs shutdownAt, but then I remembered lastTransitionTime in status.conditions, so I think fooTime has precedent!